dask/dataframe/methods.py

```diff
@@ -153,7 +153,10 @@ def describe_numeric_aggregate(stats, name=None, is_timedelta_col=False):
     assert len(stats) == 6
     count, mean, std, min, q, max = stats
 
-    typ = pd.DataFrame if isinstance(count, pd.Series) else pd.Series
+    if is_series_like(count) and hasattr(count, "to_frame"):
+        typ = type(count.to_frame())
+    else:
+        typ = type(q)
```

Comment:
Why is the hasattr needed?

Reply:
Actually this is redundant, removed the check.